src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
 // write your code here
+
+const populationElements = document.querySelectorAll('.population');
+
+const populations = Array.from(populationElements).map((pop) => {
+  return Number(pop.textContent.split(',').join(''));
+});
+
+const totalPopulation = populations.reduce((sum, population) => {
+  return sum + population;
+}, 0);
+
+const averagePopulation = totalPopulation / populations.length;
```

Comment:
Here you are calculating the average population but you are not considering the case where the populations array might be empty. This will result in 'averagePopulation' being NaN. Always check if the array is empty before performing such operations.